scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -143,10 +143,10 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
         [Target(nameof(CompileCoreHost))]
         public static BuildTargetResult PackageCoreHost(BuildTargetContext c)
         {
-            if (!string.Equals(Environment.GetEnvironmentVariable("BUILD_COREHOST_PACKAGES"), "1"))
-            {
-                return c.Success();
-            }
+//            if (!string.Equals(Environment.GetEnvironmentVariable("BUILD_COREHOST_PACKAGES"), "1"))
```

Comment:
I mainly left it commented out to make sure CI builds the packages but if all goes well I will remove these lines. I would like to get @schellap's opinion though since he was also working on this area.